src/main/java/racingcar/Names.java

```diff
@@ -0,0 +1,36 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Names {
+
+  private final ArrayList<String> names;
+
+  public Names(String unsplitted) {
+    String[] splitted = unsplitted.split(",");
+    names = new ArrayList<>(splitted.length);
+    for (String name : splitted) {
+      validate(name);
+    }
+    names.addAll(Arrays.asList(splitted));
+  }
+
+  private void validate(String name) {
+    if (name.length() > 5) {
```

Comment:
```suggestion
    if (name.length() > 최소길이) {
```
`5`매직넘버는 이름을 가질수 있지 않을까요?
다른 팀원이 보았을때,  이번경우 말고 좀 더 복잡한경우라면 `5`의 의미를 쉽게 파악할수있을까요?